.github/workflows/llvm.yml

```diff
@@ -30,8 +34,22 @@ jobs:
           key: ${{ steps.cache-key.outputs.cache-key }}
           lookup-only: true
 
-      - name: Install Tools
+      - name: Download LLVM from Rust CI
+        id: download-llvm
         if: steps.cache-llvm.outputs.cache-hit != 'true'
+        run:
+          wget https://ci-artifacts.rust-lang.org/rustc-builds/${{ steps.ls-remote.outputs.sha }}/rust-dev-nightly-${{ matrix.target }}.tar.xz && \
+            echo "download-llvm=success" << "$GITHUB_OUTPUT" || \
+            echo "download-llvm=failure" << "$GITHUB_OUTPUT"
+
+      - name: Save downloaded LLVM in cache
```

Comment:
can we just pipe from wget into tar? why do we need to save the tarball on disk?